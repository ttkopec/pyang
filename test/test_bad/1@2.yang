module one {
    namespace 'urn:one';

    prefix 'one';

    revision 2002-02-02 {
    }
}
