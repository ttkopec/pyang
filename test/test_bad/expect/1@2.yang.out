1@2.yang:1: warning: FILENAME_BAD_MODULE_NAME
1@2.yang:1: warning: FILENAME_BAD_REVISION
